source/docs/appendices/migration-guides.md

```diff
@@ -255,30 +281,31 @@ export const DesktopLeftMenu = injectIntl(
   }
 );
 ```
+
 </details>
 
 ### New style sheets
 
 If you overrode `theme/components/_components.scss` you need to add the following line to it
 
 ```scss
-@import "~theme/components/molecules/LoadingOverlay/LoadingOverlay";
-@import "~theme/components/molecules/SelectMenu/SelectMenu";
```

Comment:
This should be reverted.

Our default Prettier rule use `"`, so we should document with double quotes too.

Reply:
Hmm it's strange, where is the prettier config?